### PR TITLE
fixes: #386   Styling of Educate buttons

### LINKED ISSUE #386: Update Education button to look more like wireframes
## Summary

- [ ] Facts/Quiz buttons in the "Education" tab use our current color scheme

Look to use @acthelemann `buttons.js` styling 

## Motivation

Getting everything to a near-ready state

### PR DESCRIPTION
**Pull request by crispinamuriel**
fixes #386

### Test plan (required)

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Attached Screenshots
Screen Shot 2020-04-05 at 1 37 38 PM
Screen Shot 2020-04-05 at 1 34 27 PM

### Final Checklist

- [x ] Have you bumped the version in `package.json`?
    - Second decimal for major change, third decimal for minor change. This can go past 10 i.e. 1.0.9 !=> 1.1.0, 1.0.9 => 1.0.10
    - [Read here for more info](https://semver.org/)
- [ ] Have you added any new tests necessary?
- [ x] Is your PR rebased off the most current master?
- [ x] Have you squashed all commits, or if you will merge will you squash all commits?
- [x ] Did you use yarn, not npm?
- [] Did you use Material-UI wherever possible?
- [x ] Did you format according to Prettier?
- [x ] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

**Comment by SomeMoosery** [comment]
I feel like maybe invert the colors meaning that when you selected “Quiz” it’s filled-in blue as opposed to transparent. I think it’s transparent when you click on “Quiz” now but maybe I’m missing something? Looks good otherwise

File: client/src/components/FactQuizContainer.jsx [hunk 1]
```diff
@@ -11,7 +12,7 @@ const FactQuizContainer = () => {
   return (
     <Grid>
       <Grid>
-        <ButtonGroup size="medium" color="secondary" aria-label="outlined button group" style={{ padding: '10px' }}>
+        <ButtonGroup size="medium" color="primary" aria-label="outlined button group" style={{ ...buttonsCss.buttons }}>
```
Comment (SomeMoosery) [inline client/src/components/FactQuizContainer.jsx:15]: We shouldn't have inline styling like this - it seems like that was still in there from when we first started but we should use `makeStyles` from [Material-UI](https://material-ui.com/styles/basics/) - there should be some good example in other components here.

**Comment by SomeMoosery** [comment]
Actually one more thing I forgot to mention - need to update version in `package.json`

**Comment by crispinamuriel** [comment]
buttons both styled now, added yarn.lock
Screen Shot 2020-04-05 at 4 23 50 PM

**Comment by SomeMoosery** [comment]
Ah sorry, I may not have been clear... I was thinking one filled-in dark blue when it's selected, the other transparent - although, this is good for now. 

Still just need to bump the `version` in `/client/package.json` and it should be good to go 👍

File: client/src/components/FactQuizContainer.jsx (hunk 1 above)
Comment (SomeMoosery) [reply client/src/components/FactQuizContainer.jsx]: Thanks!

**Comment by crispinamuriel** [comment]
Okay! I changed the package.json! Thank you Carter.

**Pull request merged**